database/migrations/2024_06_07_163000_create_project_pipeline_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    public function up()
+    {
+        Schema::create('project_pipeline', function (Blueprint $table) {
+            $table->id();
+            $table->string('name', 256)->nullable();
+            $table->string('submitted_by', 256)->nullable();
+            $table->string('description', 500)->nullable();
+            $table->string('program', 256)->nullable();
+            $table->string('cohort', 256)->nullable();
+            $table->string('publish_for', 256)->nullable();
```

Comment:
Yes, the purpose of the `submitted_by` field is to store the user's name for the moment. Would it be relevant to associate it as a foreign key?

The `publish_for` field is to store a URL.

Reply:
Yeah, it would be better to associate it to the user record and pull the name from there when needed in an API response. Storing the user's name directly is pretty fragile - what if the user record name changes?